cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityPrivilegesTest.scala

```diff
@@ -224,20 +232,34 @@ class SecurityPrivilegesTest extends DocumentingTest with QueryStatisticsTestSup
 
       p("An example usage of the `REVOKE` command is given here:")
       query("REVOKE GRANT TRAVERSE ON GRAPH neo4j NODES Post FROM regularUsers", ResultAssertions((r) => {
-        // TODO: revoke a privilege that exists
-        assertStats(r, systemUpdates = 0)
+        assertStats(r, systemUpdates = 1)
       })) {
         statsOnlyResultTable()
       }
       p(
         """While it can be explicitly specified that revoke should remove a `GRANT` or `DENY`, it is also possible to revoke either one by not specifying at all as the next example demonstrates.
           |Because of this, if there happen to be a `GRANT` and a `DENY` on the same privilege, it would remove both.""".stripMargin)
       query("REVOKE TRAVERSE ON GRAPH neo4j NODES Payments FROM regularUsers", ResultAssertions((r) => {
-        // TODO: revoke a privilege that exists
-        assertStats(r, systemUpdates = 0)
+        assertStats(r, systemUpdates = 2)
       })) {
         statsOnlyResultTable()
       }
+
+      p(
+        """Some privileges are compound privileges and contains sub-privileges, for example <<administration-security-administration-database-indexes, `INDEX MANAGEMENT`>> which covers `CREATE INDEX` and `DROP INDEX`.
```

Comment:
Should we mention that you can't revoke a sub-privilege when a compound version of the compound privilege exists?

Reply:
The tricky part is that in theory all compound privileges should have been split to leaves on creation. This is true for all that were created using Cypher, but not true for some of those created in the system graph initialization. So this problem is specific to system graph initialization privileges only, and perhaps needs to be described in the operations manual, not here where we focus on Cypher commands only (and there are no bugs).

Reply:
`This is true for all that were created using Cypher` Except for ROLE MANAGEMENT :P